drivers/cmd/web/main_test.go

```diff
@@ -50,6 +50,33 @@ func TestSendRequest(t *testing.T) {
 	response := <-request.Reply
 	assert.Equal(t, "200", response.Payload.(map[string]interface{})["status_code"])
 	assert.NotContains(t, response.Labels, "error")
+	foo, _ := response.Payload.(map[string]interface{})["json"].(map[string]interface{})["foo"]
+	assert.Equal(t, "bar", foo, "JSON data miss-match")
```

Comment:
Should be mismatch. And maybe something a little more descriptive than `foo` here?